Documentation/ceph-cluster-crd.md

```diff
@@ -138,6 +138,7 @@ Below are the settings available, both at the cluster and individual node level,
   - `^[^r]`: Selects all devices that do *not* start with `r`
 - `devices`: A list of individual device names belonging to this node to include in the storage cluster.
   - `name`: The name of the device (e.g., `sda`).
+  - `fullpath`: The full path to the device (e.g., `/dev/disk/by-id/scsi-35000c5008335c83f`). If specified then `name` can be omitted.
```

Comment:
I'd suggest to leave out the ceph changes with this PR. Ceph will require more in-depth changes as seen in #2845. Let's go ahead with this to get it working for edgefs, then we should get the ceph PR rebased on this.

Reply:
done